anguish/zone_status.lua

```diff
@@ -18,45 +18,56 @@
 -- Zone Trash:
 --
 --]]
-local lockout_bit;
 local instance_id = 0;
-local qglobals = {};
-local charid_list;
-local current_bit = 0;
-local entity_list;
-local spawn_orb=false;
-local instance_requests = require("instance_requests");
 local Anguish_Lockouts = {};
 
-function setup_lockouts()
+local KELDOVAN_EVENT    = "Keldovan the Harrier"
+local JELVAN_EVENT      = "Rescuing Jelvan"
+local TURE_EVENT        = "Ture"
+local HANVAR_EVENT      = "Warden Hanvar"
+local AMV_EVENT         = "Arch Magus Vangl"
+local OMM_EVENT         = "Overlord Mata Muram"
+local LOWER_GLOBE_EVENT = "Lower Globe of Discordant Energy"
+local UPPER_GLOBE_EVENT = "Upper Globe of Discordant Energy"
+local REPLAY_EVENT      = "Replay Timer"
+
+function setup_lockouts(expedition)
 	Anguish_Lockouts = {
-		[317005] = {'Anguish_keldovan', 1,   Spawn_keldovan},
-		[317004] = {'Anguish_jelvan',   2,   Spawn_jelvan},
-		[317003] = {'Anguish_ture',     4,   Spawn_ture},
-		[317002] = {'Anguish_hanvar',   8,   Spawn_hanvar},
-		[317107] = {'Anguish_amv',      16,  PH_amv},
-		[317109] = {'Anguish_omm',      32,  Spawn_omm},
-		[1]		 = {'Anguish_lower_orb',64,  PH_lorb},
-		[2]		 = {'Anguish_upper_orb',128, PH_uorb},
-		[3]		 = {'Anguish_augs'	   ,256, PH_augs}
+		[317005] = {KELDOVAN_EVENT,    eq.seconds("4d12h"), Spawn_keldovan},
```

Comment:
What's with the different numbered keys? Are they supposed to be corresponding to a database value ?

Reply:
Those are npc ids for lookup from the original script (I tried to preserve most of the original script for this one; it was my original test bed for the source)